modules/terraform-zscc-asg-aws/main.tf

```diff
@@ -81,8 +71,10 @@ resource "aws_autoscaling_group" "cc_asg" {
   vpc_zone_identifier       = distinct(var.cc_subnet_ids)
   max_size                  = var.max_size
   min_size                  = var.min_size
-  health_check_type         = "ELB"
+  health_check_type         = var.health_check_type
   health_check_grace_period = var.health_check_grace_period
+  default_instance_warmup   = var.instance_warmup

```

Comment:
@anichari

Autoscaling changes:
- health_check_type changed from ELB to EC2 - CHANGE
- default_instance_warmup w/ default 900 - ADD
-